src/surge-xt/osc/OpenSoundControl.cpp

```diff
@@ -119,6 +126,7 @@ void OSCListener::oscMessageReceived(const juce::OSCMessage &message)
         if (!message[0].isFloat32())
         {
             // Not a valid data value
+            std::cout << "Invalid data type (not float)." << std::endl;
```

Comment:
If we actually want to report this to the user, storage->reportError is the way to go. If we don't then probably strip the print.

Reply:
Or wrap in a #debug

Reply:
Wrapped it in #debug.